frontend/src/libs/components/app/app.tsx

```diff
@@ -37,7 +37,7 @@ const App: React.FC = () => {
   return (
     <Loader isLoading={isLoading}>
       <>
-        <Header user={null} />
+        {hasUser && <Header user={user} />}
```

Comment:
I believe it's not necessary if we have separate check that user exists inside header component
So lets deal with a single approach - we are checking if user exists here OR inside header component (not in both places)